main_flow/main_flow.py

```diff
@@ -87,15 +87,15 @@ def periodically_save_cache():
 
 def steamimfy(table):
     steamedTable = copy.deepcopy(table)
-
-    for row in steamedTable:
-        steam_list_of_words_with_cache(row)
+    steam_list_of_words_with_cache(steamedTable)
     return steamedTable
 
 
-def steam_list_of_words_with_cache(row):
-    words = row[0].replace('\n', ' ').replace('\r', ' ').replace(',', ' '). \
-        replace('.', ' ').replace(':', ' ').replace('!', ' ').replace('?', ' ').split(' ')
+def steam_list_of_words_with_cache(steamedTable):
+    words = []
+    for row in steamedTable:
+        words += row[0].replace('\n', ' ').replace('\r', ' ').replace(',', ' '). \
+            replace('.', ' ').replace(':', ' ').replace('!', ' ').replace('?', ' ').split(' ')
```

Comment:
please  extract to function with a meaningful name so the code will be more readable